go/bufio2/bufio_test.go

```diff
@@ -292,7 +292,7 @@ func TestUnreadRuneError(t *testing.T) {
 	if err != nil {
 		t.Error("unexpected error on ReadRune (2):", err)
 	}
-	for _ = range buf {
+	for range buf {
```

Comment:
This syntactic form was added in Go 1.4. Are we ok with dropping support for 1.3 and below?